registration/auth_backends.py

```diff
@@ -50,5 +50,6 @@ def authenticate(self, username=None, password=None):
             # difference between an existing and a non-existing user (#20760).
             UserModel().set_password(password)
         else:
-            if user.check_password(password):
+            if user.check_password(password) and self.user_can_authenticate(user):
```

Comment:
What prompted this change?

Reply:
This was updated to reflect django 2.2.2's base [authenticate](https://github.com/django/django/blob/9400c96b209855d7b97e3223e4b8ba7751c357cc/django/contrib/auth/backends.py#L16-L27) method. I'm not sure how ocim manages disabling users, but I thought it best to update it and discuss in the review here.

Reply:
@clemente are you aware if this may cause issues? Is there ever a case where this function may return false but we still want the user to be able to login?

Reply:
No problem here. [user_can_authenticate](https://github.com/django/django/blob/4f7328ce8a35160d155c41d362c3d674f8ef4d2d/django/contrib/auth/backends.py#L54) mainly checks if `user.is_active`. If a user isn't active, they shouldn't log in.